schemas/connections.schema.json

```diff
@@ -414,18 +414,50 @@
             "cacert": {
               "description": "CA certificate file to verify with",
               "type": "string"
+            },
+            "socket_timeout": {
+              "description": "Socket timeout in seconds",
+              "type": "number",
+              "minimum": 0,
+              "default": 10.0
+            },
+            "socket_connect_timeout": {
+              "description": "Socket connection timeout in seconds",
+              "type": "number",
+              "minimum": 0,
+              "default": 2.0
+            },
+            "socket_keepalive": {
+              "description": "Enable/disable TCP keepalive",
+              "type": "boolean",
+              "default": true
+            },
+            "health_check_interval": {
+              "description": "Interval for health checks in seconds",
+              "type": "number",
+              "minimum": 0,
+              "default": 60
             }
           },
           "additionalProperties": false,
```

Comment:
do we have extra_args? I thought we have it for other DBs.

Reply:
In other DBs we do, but this, since it's a Redis connection identical to the one that is used in RDI, we have those properties in the connection level.